### ISSUE TITLE
Screen Reader: Many Pages: The field of distance is announced without a currency

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74871]
## Action Performed
Prerequisites:
- The user is signed in.

1. Using Chrome + JAWS, open the URL - https://new.expensify.com/
2. Navigate to the "+" icon and press Enter
3. Tab to the "Track distance" menu item and press Enter to open the Pop-up
4. Tab to the "Manual" Tab and press Enter to select
5. Tab to the "Distance" field and add input to the field
6. Observe the screen reader announcement

Other Occurrences:
1. On Send Invoice
2. On Create Expense

## Expected Result
When the field of the distance receives focus, the input field should have a programmatic name announced, e.g.:
- 'distance (mi)' or similar

## Actual Result
When the field of the distance receives focus, the screen reader announces the field as “0 edit type in text”; no programmatic name is announced, so that the users cannot identify what information has to be entered.

HTML:
<div dir="auto" class="css-146c3p1" style="color: rgb(175, 187, 176); font-size: 40px; text-align: left; font-family: &quot;Expensify New Kansas&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 500;">mi</div>

## Area issue was found
Track Distance

## Failed Checkpoint
WCAG 1.3.1 Info and Relationships
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships

## User Impact
Users with different disabilities employ assistive technology to adapt content according to their specific needs. For example, users who are blind (using a screen reader) benefit when information conveyed through color is also available in text (including text alternatives for images that use color to convey information), and users who are deaf-blind and use braille (text) refreshable displays may be unable to access color-dependent information.

## Suggested Resolution
Add a visible name and associate it with the field or use a visually hidden programmatic name, e.g.:
- 'distance (mi)' or similar
https://www.w3.org/WAI/tutorials/forms/labels/

## Environment(s)
Windows Windows 11 Home - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/942305b5-6991-4725-836a-e80e18fe2dd3

<img width="1919" height="969" alt="Image" src="https://github.com/user-attachments/assets/335d456a-3a5b-46d9-bf02-ade1e1b8358f" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6997121
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by mallenexpensify** [comment]
Accessibility, unassigning, will be addressed in bulk once all issues are created.
